### ISSUE TITLE
Landmark Generator

### ISSUE THREAD

**Issue by KhalilWong** [KylinYee/R2-Talker-code#6]
If I need to use a custom audio wav file to synthesize a video, is it only possible with "landmark generator"? Also, when is the "landmark generator" expected to be released? Thanks!

**Comment by KylinYee** [comment]
Yes, you must use the landmark generator. We plan to release the landmark generator before March. If necessary, you can temporarily use the landmark generator in the [geneface](https://github.com/yerfor/GeneFace) .

**Comment by KhalilWong** [comment]
> Yes, you must use the landmark generator. We plan to release the landmark generator before March. If necessary, you can temporarily use the landmark generator in the [geneface](https://github.com/yerfor/GeneFace) .

How can i use the landmark generator in the geneface with audio rather than video as its input?

**Comment by xiaoxiongzhg** [comment]
> Yes, you must use the landmark generator. We plan to release the landmark generator before March. If necessary, you can temporarily use the landmark generator in the [geneface](https://github.com/yerfor/GeneFace) .

Hello author, I know it’s a bit offensive to ask this, when will the code be released?

**Comment by kike-0304** [comment]
> > 是的，您必须使用地标生成器。我们计划在三月份之前发布地标生成器。[如果需要，您可以暂时使用geneface](https://github.com/yerfor/GeneFace)中的地标生成器。
> 
> 我如何在geneface中使用音频而不是视频作为输入的地标生成器？

请问您解决这个问题了吗，音频怎么用geneface生成landmark？

**Comment by KhalilWong** [comment]
还没有，再等等看吧
